packages/votingV2/src/utils/constants.ts

```diff
@@ -4,7 +4,7 @@ import { toDecimal } from "./decimals";
 export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
 export const STORE_ID = "0x54f44ea3d2e7aa0ac089c4d8f7c93c27844057bf";
 export const GOVERNOR_ADDRESS_STRING = "0x592349f7dedb2b75f9d4f194d4b7c16d82e507dc";
-export let VOTING_TOKEN_ADDRESS = Address.fromString("0x04fa0d235c4abf4bcf4787af4cf447de572ef828");
+export let VOTING_TOKEN_ADDRESS = Address.fromString("0xBa950974AFdae520cEE596c98295150Cf8D9659c");
```

Comment:
it seems kinda strange that we store these here vs a const file with all the addresses in one place. I know this was a carry over from the previous implementation but we should consider cleaning this up a bit.